### ISSUE TITLE
Migrating from libgnome-keyring to Secret API

### ISSUE THREAD

**Issue by GoogleCodeExporter** [yendo/gphotoframe#102]
```
freedesktop.org - Specifications/secret-storage-spec
http://www.freedesktop.org/wiki/Specifications/secret-storage-spec

```

Original issue reported on code.google.com by `yendo0206` on 21 Dec 2011 at 4:12

**Comment by GoogleCodeExporter** [comment]
Original comment by `yendo0206` on 28 Dec 2011 at 6:41
- Changed state: **Started**

**Comment by GoogleCodeExporter** [comment]
```
Implemented in revision a75c6a939287.
```

Original comment by `yendo0206` on 15 Feb 2012 at 3:23
- Changed state: **Fixed**
